llvm/include/llvm/TargetParser/Triple.h

```diff
@@ -366,14 +366,26 @@ class Triple {
   /// @name Normalization
   /// @{
 
+  /// Canonical form
+  enum class CanonicalForm {
+    NONE = 0,
```

Comment:
```suggestion
    ANY = 0,
```

This might be clearer